lib/perf_hooks.js

```diff
@@ -108,13 +115,19 @@ ObjectDefineProperties(Performance.prototype, {
   }
 });
 
+const toJSON = () => {
+  const obj = new InternalPerformance();
+  return obj.toJSON();
+};
+
 module.exports = {
   PerformanceEntry,
   PerformanceMark,
   PerformanceObserver,
   monitorEventLoopDelay,
   createHistogram,
   performance: new InternalPerformance(),
+  toJSON
```

Comment:
I'm not sure this should exist?

Reply:
Yes I also think so since internal performance object already have tojson method. Will remove it.